### ISSUE TITLE
Crash in CLI created app starting from 0.60.0-vnext.127

### ISSUE THREAD

**Issue by ddalp** [microsoft/react-native-windows#3975]
Does not repro on vnext.126

Most likely caused by this latest commit: https://github.com/microsoft/react-native-windows/commit/b66b32813dbf25e6bf201d9b35f5833eca3588a4

"array subscript out of range"

>	Microsoft.ReactNative.dll!std::array<enum YGEdge,4>::operator[](unsigned __int64 _Pos) Line 184	C++
 	Microsoft.ReactNative.dll!YGNode::getTrailingBorder(const YGFlexDirection flexDirection) Line 466	C++
 	Microsoft.ReactNative.dll!YGNode::getTrailingPaddingAndBorder(const YGFlexDirection axis, const float widthSize) Line 516	C++
 	Microsoft.ReactNative.dll!YGNodePaddingAndBorderForAxis(const YGNode * const node, const YGFlexDirection axis, const float widthSize) Line 991	C++
 	Microsoft.ReactNative.dll!YGNodelayoutImpl(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2742	C++
 	Microsoft.ReactNative.dll!YGLayoutNodeInternal(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, const char * reason, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 3832	C++
 	Microsoft.ReactNative.dll!YGNodeComputeFlexBasisForChild(YGNode * const node, YGNode * const child, const float width, const YGMeasureMode widthMode, const float height, const float ownerWidth, const float ownerHeight, const YGMeasureMode heightMode, const YGDirection direction, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 1377	C++
 	Microsoft.ReactNative.dll!YGNodeComputeFlexBasisForChildren(YGNode * const node, const float availableInnerWidth, const float availableInnerHeight, YGMeasureMode widthMeasureMode, YGMeasureMode heightMeasureMode, YGDirection direction, YGFlexDirection mainAxis, YGConfig * const config, bool performLayout, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 1890	C++
 	Microsoft.ReactNative.dll!YGNodelayoutImpl(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2793	C++
 	Microsoft.ReactNative.dll!YGLayoutNodeInternal(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, const char * reason, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 3832	C++
 	Microsoft.ReactNative.dll!YGNodeComputeFlexBasisForChild(YGNode * const node, YGNode * const child, const float width, const YGMeasureMode widthMode, const float height, const float ownerWidth, const float ownerHeight, const YGMeasureMode heightMode, const YGDirection direction, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 1377	C++
 	Microsoft.ReactNative.dll!YGNodeComputeFlexBasisForChildren(YGNode * const node, const float availableInnerWidth, const float availableInnerHeight, YGMeasureMode widthMeasureMode, YGMeasureMode heightMeasureMode, YGDirection direction, YGFlexDirection mainAxis, YGConfig * const config, bool performLayout, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 1890	C++
 	Microsoft.ReactNative.dll!YGNodelayoutImpl(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2793	C++
 	Microsoft.ReactNative.dll!YGLayoutNodeInternal(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, const char * reason, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 3832	C++
 	Microsoft.ReactNative.dll!YGDistributeFreeSpaceSecondPass(YGCollectFlexItemsRowValues & collectedFlexItemsValues, YGNode * const node, const YGFlexDirection mainAxis, const YGFlexDirection crossAxis, const float mainAxisownerSize, const float availableInnerMainDim, const float availableInnerCrossDim, const float availableInnerWidth, const float availableInnerHeight, const bool flexBasisOverflows, const YGMeasureMode measureModeCrossDim, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2168	C++
 	Microsoft.ReactNative.dll!YGResolveFlexibleLength(YGNode * const node, YGCollectFlexItemsRowValues & collectedFlexItemsValues, const YGFlexDirection mainAxis, const YGFlexDirection crossAxis, const float mainAxisownerSize, const float availableInnerMainDim, const float availableInnerCrossDim, const float availableInnerWidth, const float availableInnerHeight, const bool flexBasisOverflows, const YGMeasureMode measureModeCrossDim, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2309	C++
 	Microsoft.ReactNative.dll!YGNodelayoutImpl(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 2915	C++
 	Microsoft.ReactNative.dll!YGLayoutNodeInternal(YGNode * const node, const float availableWidth, const float availableHeight, const YGDirection ownerDirection, const YGMeasureMode widthMeasureMode, const YGMeasureMode heightMeasureMode, const float ownerWidth, const float ownerHeight, const bool performLayout, const char * reason, YGConfig * const config, YGMarkerLayoutData & layoutMarkerData, void * const layoutContext) Line 3832	C++
 	Microsoft.ReactNative.dll!YGNodeCalculateLayoutWithContext(YGNode * node, float ownerWidth, float ownerHeight, YGDirection ownerDirection, void * layoutContext) Line 4056	C++
 	Microsoft.ReactNative.dll!YGNodeCalculateLayout(YGNode * node, float ownerWidth, float ownerHeight, YGDirection ownerDirection) Line 4161	C++
 	Microsoft.ReactNative.dll!react::uwp::NativeUIManager::DoLayout() Line 851	C++

**Comment by acoates-ms** [comment]
I think this is what the ENUM_BITFIELDS_NOT_SUPPORTED define was for.  Its part of folly.

**Comment by NickGerleman** [comment]
@acoates-ms ENUM_BITFIELDS_NOT_SUPPORTED doesn't seem to be in folly, but we do use it in microsoft/react-native.

I would expect we still define the definition though, given the only code that checks it should define it if it sees MSVC. The upstream fix for this made it into 0.60.2, so I think we should be fine when using upstream.
```
// Until we sync past this: https://github.com/facebook/react-native/commit/a7e4ce0f0d83b6162d77db7d680e838666f6d938#diff-140c478f1096be49a1b121931a7680ee
// We have to turn off the bitfield usage for MSVC builds
#ifdef _MSC_VER
#define ENUM_BITFIELDS_NOT_SUPPORTED
#endif

#if !defined(ENUM_BITFIELDS_NOT_SUPPORTED)
#define BITFIELD_ENUM_SIZED(num) : num
#else
#define BITFIELD_ENUM_SIZED(num)
#endif
```
